packages/terra-menu/src/Menu.module.scss

```diff
@@ -46,6 +46,10 @@
       word-wrap: normal;
     }
 
+    .main-header-title {
+      margin-left: var(--terra-menu-header-title-margin-left, 10px);
```

Comment:
Since fusion theme is ported to terra-menu, I think you can add this variable in [orion-fusion-theme.scss](https://github.com/cerner/terra-framework/blob/master/packages/terra-menu/themes/orion-fusion-theme/orion-fusion-theme.scss) and might need to plan for add in other themes as well.

Reply:
updated here ef3239cd78e5d657c848f92d318f790c6a31f47a